### ISSUE TITLE
Icon-Set 1 incomplete

### ISSUE THREAD

**Issue by scooty99** [rg-engineering/ioBroker.daswetter#81]
If you choose the Icon-Set No. 1 in the Instance-Settings you have 19 images available.
But the adapter currently requests also an icon 20

Example:
Object: daswetter.0.NextDays.Location_1.Day_2.iconURL
Value: /adapter/daswetter/icons/tiempo-weather/galeria1/20.png

So, VIS just shows an error instead of an image.

**Comment by xolom** [comment]
Same problem.. 20 and 21 is missing

**Comment by DerMaulwurf** [comment]
There is not much the devs can do about that, I guess.
I just checked the icon-pack at https://www.daswetter.com/documentacion_api/iconos.zip.
Galeria1 to galeria4 each contain only 19 icons. Galeria5 contains 22 icons.

Additionaly I noticed that the icons in galeria1 display different conditions compared to icons in galeria5.
1.pgn to 16.png are the same, 
17.png is like 20.pgn
18.png is like 21.png
The rest of the icons are missing from galeria1 to galeria4.
